oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -200,3 +203,100 @@ def _get_oneview_resource(uuid):
                 raise  # Raise any unexpected errors
 
     raise OneViewRedfishError("Could not find computer system with id " + uuid)
+
+
+@computer_system.route(ComputerSystem.BASE_URI + "/", methods=["POST"])
+def create_composed_system():
+    if not request.is_json:
+        abort(status.HTTP_400_BAD_REQUEST,
+              "The request content should be a valida JSON")
+
+    body = request.get_json()
+
+    #  TODO(@ricardogpsf) use redfish validation instead of this intern class
+    class ComposedSystem(RedfishJsonValidator):
+        SCHEMA_NAME = 'ComputerSystem'
+
+        def __init__(self, composed_system):
+            super().__init__(self.SCHEMA_NAME)
+
+            self.redfish = composed_system
+
+        def validate(self):
+            self._validate()
+
+    try:
+        resource = ComposedSystem(body)
+        resource.validate()
+
+        blocks = resource.redfish["Links"]["ResourceBlocks"]
+        block_ids = \
+            list(map(lambda b: b["@odata.id"].split("/")[-1], blocks))
```

Comment:
Nice!

Reply:
I have saw with attention now, I'll do this better without use the map or list methods. ;)